validation/validation.go

```diff
@@ -0,0 +1,72 @@
+package validation
+
+import (
+	"strconv"
+
+	"fmt"
+	roles "github.com/hpcloud/fissile/model"
+)
+
+// ValidateNonnegativeField validates that given value is not negative.
+func ValidateNonnegativeField(value int64, field string) ErrorList {
+	allErrs := ErrorList{}
+	if value < 0 {
+		allErrs = append(allErrs, Invalid(field, value, `must be greater than or equal to 0`))
+	}
+	return allErrs
+}
+
+// ValidatePort validates that given value is valid and in range 0 - 65535.
+func ValidatePort(port string, field string) ErrorList {
+	allErrs := ErrorList{}
+
+	portInt, err := strconv.Atoi(port)
```

Comment:
Alternatively, `strconv.ParseInt(port, 10, 16)` should do both conversion and range checking.  On the other hand, the error message is not as nice… so I guess this is actually fine.